tests/kwlib_test.sh

```diff
@@ -355,4 +356,21 @@ function statistics_manager_Test
   assertTrue "($ID) Database day" '[[ ! -f "$statistics_path/$this_year_and_month/$today" ]]'
 }
 
+function command_exists_Test
+{
+  local ID
+  local fake_command="a-non-existent-command -p"
+  local real_command="mkdir"
+
+  ID=1
```

Comment:
We want to avoid the ID variable now. You can drop it from this test and replace all occurrences of `$ID` with `$LINENO`.